config/database.yml

```diff
@@ -86,11 +86,8 @@ test:
 # You can use this database configuration with:
 #
 #   production:
-#     url: <%= ENV['DATABASE_URL'] %>
+#     url: <%= ENV.fetch("DATABASE_URL") %>
 #
 production:
   <<: *default
-  database: PracticalDeveloper_production
-  pool: <%= ENV['DATABASE_POOL_SIZE'] %>
```

Comment:
Why did you choose to get rid of the pool size here for production?

Reply:
Production extends the default config in database.yml which has the pool configured or deafults to 5.
Previously (current master) has two config variables to effect pool size.
The default config `pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>`
and production config `pool: <%= ENV['DATABASE_POOL_SIZE'] %>` which if null would default to RAILS_MAX_THREADS or 5.

The new config in default inherited by production is trying to maintain the same behaviour with 
`pool: <%= ENV.fetch("DATABASE_POOL_SIZE") { ENV.fetch("RAILS_MAX_THREADS", 5).to_i `